src/manifest/MsQuicEtw.man

```diff
@@ -2599,6 +2614,33 @@
               template="tid_CONN_SERVER_RESUME_TICKET"
               value="5175"
               />
+          <event
+              keywords="ut:Connection"
+              level="win:Informational"
+              message="$(string.Etw.ConnClientCompatibleVersionList)"
+              opcode="Connection"
+              symbol="QuicConnClientCompatibleVersionList"
+              template="tid_CONN_VNL"
+              value="5176"
+              />
+          <event
+              keywords="ut:Connection"
+              level="win:Informational"
+              message="$(string.Etw.ConnClientReceivedVersionList)"
+              opcode="Connection"
+              symbol="QuicConnClientReceivedVersionList"
+              template="tid_CONN_VNL"
+              value="5177"
+              />
+          <event
+              keywords="ut:Connection"
+              level="win:Informational"
+              message="$(string.Etw.ConnServerSupportedVersionList)"
+              opcode="Connection"
+              symbol="QuicConnServerSupportedVersionList"
+              template="tid_CONN_VNL"
+              value="5178"
+              />
```

Comment:
These can probable have the `ut:LowVolume` keywords as well, as they shouldn't be particular common.

Reply:
There'll be 2 per connection, at most (server side and client side). Is that low volume?

Reply:
Yeah.